src/react-components/src/components/03-organisms/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+import Accordion from "./Accordion";
+import { action } from "@storybook/addon-actions";
+import { text } from "@storybook/addon-knobs";
+import Checkbox from "../../01-atoms/Forms/Checkbox";
+import UnorderedList from "../../03-organisms/List/UnorderedList";
+
+
+export default {
+  title: "Accordion",
+  component: Accordion
+};
+const checkboxes = [
+  <Checkbox onChange={action("changed")} checkboxId="checkbox-1" labelOptions={{ id: "checkbox-1-label", labelContent: <>Checkbox 1</> }}></Checkbox>,
+  <Checkbox onChange={action("changed")} checkboxId="checkbox-2" labelOptions={{ id: "checkbox-1-label", labelContent: <>Checkbox 2</> }}></Checkbox>,
+  <Checkbox onChange={action("changed")} checkboxId="checkbox-3" labelOptions={{ id: "checkbox-1-label", labelContent: <>Checkbox 3</> }}></Checkbox>
```

Comment:
This is bad because it's showing the same ID as input and it creates an a11y issue:
<img width="507" alt="Screen Shot 2020-04-12 at 5 11 03 PM" src="https://user-images.githubusercontent.com/1280564/79080039-a91d2700-7ce0-11ea-86dd-fc9c5d0e7ceb.png">